### PR TITLE
upgrade spark to 2.4.6

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/sql/benchmark/TPCHQuerySuite.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ class TPCHQuerySuite extends BaseTiSparkTest {
     "q12",
     "q13",
     "q14",
-    "q15",
+    //"q15",
```
Comment (birdstorm) [inline core/src/test/scala/org/apache/spark/sql/benchmark/TPCHQuerySuite.scala:41]: issue link?

Comment (marsishandsome) [reply core/src/test/scala/org/apache/spark/sql/benchmark/TPCHQuerySuite.scala]: done

**Review by birdstorm — APPROVED** [review]
LGTM

**Pull request merged**